core/kazoo_modb-1.0.0/priv/couchdb/views/notifications.json

```diff
@@ -3,7 +3,7 @@
     "language": "javascript",
     "views": {
         "smtp_log": {
-            "map": "function(doc) {if (doc.pvt_deleted || doc.pvt_type != 'notify_smtp_log') return; emit(doc.pvt_created);} ;"
+            "map": "function(doc) { if (doc.pvt_deleted || doc.pvt_type != 'notify_smtp_log') return; emit(doc.pvt_created, {'id': doc._id, 'from': doc.from, 'to': doc.to, 'created': doc.pvt_created}); }"
```

Comment:
Shouldn't you include `doc.error` in the view value? Fax's `smtp_log` view has an error property.